components/support/migration/src/main/java/mozilla/components/support/migration/AbstractMigrationService.kt

```diff
@@ -75,29 +78,61 @@ abstract class AbstractMigrationService : Service() {
         scope.cancel()
     }
 
-    private fun showForegroundNotification() {
-        val channel = ensureChannelExists()
+    private fun showStatusNotification(isInProgress: Boolean) {
```

Comment:
nit: let's keep the method as-is and create a separate method for showing the completion notification. that will prevent all those `if/else` statements here.